docs/gl_objects/iterations.rst

```diff
@@ -31,3 +31,13 @@ List iterations for a project's ancestor groups::
 List iterations for a group::
 
     iterations = group.iterations.list()
+
+Unavailable filters or filters conflicts::
+    
+    In case you are trying to pass a parameter that is not supported or
+    collides with a python keyword (i.e. `in`) you have to use the
+    `query_parameters` argument:
```

Comment:
There aren't really "unsupported" filters (since we just pass whatever the user provides to the API), only those that clash with other keywords. I'd maybe phrase it like this:
```suggestion
    In case you are trying to pass a parameter that collides with a python
    keyword (i.e. `in`) or with python-gitlab's internal arguments, you'll have
    to use the `query_parameters` argument:
```